pkg/cli/logfile/logfile_unix.go

```diff
@@ -101,6 +101,7 @@ func Hook() error {
 
 		cmd.Stdout = io.MultiWriter(os.Stdout, rec)
 		cmd.Stderr = io.MultiWriter(os.Stderr, rec)
+		cmd.Stdin = os.Stdin
```

Comment:
The error was being triggered on line 84, so this isn't fixing the bug you think it is.